src/main/java/com/snowflake/kafka/connector/IConnectorConfigValidator.java

```diff
@@ -0,0 +1,14 @@
+package com.snowflake.kafka.connector;
+
+import java.util.Map;
+
+public interface IConnectorConfigValidator {
```

Comment:
do we need to follow here the "I" interface naming convention? I would just do as usual with the implementation having postfix Impl or prefix Main/Default

Reply:
Changed to Default prefix. We should align to one convention as right now we have no interfaces = no convention ;)